tests/test_jetpack.py

```diff
@@ -176,34 +166,32 @@ def test_pass_manifest_elements():
                      "jsSHA256": bootstrap_hash,
                      "docsSHA256": bootstrap_hash}}}
 
-    err = _do_test(MockXPI({"bootstrap.js": bootstrap,
-                            "harness-options.json":
-                                json.dumps(harnessoptions),
-                            "resources/bootstrap.js": bootstrap}))
-    print err.print_summary(verbose=True)
+    with accepted_version("frameworks", "jetpack", "versions", "1.7"):
+        err = _do_test(MockXPI({"bootstrap.js": bootstrap,
+                                "harness-options.json": harnessoptions,
+                                "resources/bootstrap.js": bootstrap}))
     assert not err.failed()
     assert "jetpack_loaded_modules" in err.metadata
-    nose.tools.eq_(err.metadata["jetpack_loaded_modules"],
-                   ["addon-kit-lib/drawing.js"])
-    assert "jetpack_identified_files" in err.metadata
-    assert "identified_files" in err.metadata
-    assert "bootstrap.js" in err.metadata["jetpack_identified_files"]
-    assert "bootstrap.js" in err.metadata["identified_files"]
+    eq_(err.metadata["jetpack_loaded_modules"],
+        ["addon-kit-lib/drawing.js"])
+    assert_framework_files(err, ["bootstrap.js"])
 
-    assert "jetpack_unknown_files" in err.metadata
-    assert not err.metadata["jetpack_unknown_files"]
+    if False:
```

Comment:
Yes please ;) I'd rather have commented out code (even though I dislike it a lot, if it's commented, it's not used, if it's not used, it shouldn't be there) than this construct.